tmk_core/protocol/arm_atsam/main_arm_atsam.c

```diff
@@ -160,6 +155,75 @@ void send_consumer(uint16_t data)
 #endif //EXTRAKEY_ENABLE
 }
 
+uint8_t g_drvid;
+
+void main_subtask_usb_state(void)
+{
+    if (usb_state == USB_STATE_POWERDOWN)
+    {
+        uint32_t timer_led = timer_read32();
+
+        led_on;
+        if (led_enabled)
+        {
+            for (g_drvid=0;g_drvid<ISSI3733_DRIVER_COUNT;g_drvid++)
+            {
+                I2C3733_Control_Set(0);
+            }
+        }
+        while (usb_state == USB_STATE_POWERDOWN)
+        {
+            if (timer_read32() - timer_led > 1000) led_off; //Good to indicate went to sleep, but only for a second
+        }
+        if (led_enabled)
+        {
+            for (g_drvid=0;g_drvid<ISSI3733_DRIVER_COUNT;g_drvid++)
```

Comment:
Add whitespace between the for expressions and around binary operators. (multiple occurrences in this file)